mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/vault/attachements/AttachmentsPresenter.kt

```diff
@@ -56,6 +56,7 @@ class AttachmentsPresenter(var view: IAttachmentsPresenter.IView?) :
     @SuppressLint("CheckResult")
     override fun importVaultFiles(uris: List<Uri>, parentId: String?, deleteOriginal: Boolean) {
         if (uris.isEmpty()) return
```

Comment:
If uris is Empty we shouldn't even fire the event to show import Bottom sheet, so why we are using this `postValue`?